rustviz_mdbook/view_examples.sh

```diff
@@ -24,33 +24,37 @@ printf "Generating visualizations for the following examples: \n"
 
 # Uncomment the examples are being tested
 declare -a targetExamples=(
-    "copy"
-    "func_take_ownership"
-    "func_take_return_ownership"
-    "function"
-    "hatra1"
-    "hatra1_test"
-    "hatra2"
-    "immutable_borrow"
-    "immutable_borrow_lifetime"
-    "immutable_borrow_method_call"
-    "immutable_variable"
-    "move_assignment"
-    "move_different_scope"
-    "move_func_return"
-    "multiple_immutable_borrow"
-    "mutable_borrow"
-    "mutable_borrow_method_call"
-    "mutable_variables"
-    "nll_lexical_scope_different"
-    "printing"
-    "string_from_move_print"
-    "string_from_print"
-    "struct_lifetime"
-    "struct_rect"
-    "struct_rect2"
-    "struct_string"
-    "extra_credit"
+    # "copy"
```

Comment:
why were all these examples deleted / commented out? don't want that in the base branch...

Reply:
will uncomment those. I commented those for the convenience of testing all lifetime vis funtionalities.